package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "npm-check",
   "version": "6.0.1",
+  "type": "module",
   "description": "Check for outdated, incorrect, and unused dependencies.",
   "main": "lib",
```

I think that we should use the new `exports` key together with `type: module`:

```suggestion
  "exports": "lib/index.js",
```